config/desktop/jammy/appgroups/net-tools/packages

```diff
@@ -1,3 +1,18 @@
-gkrellm
```

Comment:
I see the next 2 got moved to a different file, but I have not found `gkrellm` anywhere yet